deebot_client/commands/xml/common.py

```diff
@@ -25,7 +25,6 @@ class XmlCommand(Command):
     data_type: DataType = DataType.XML
 
     @property  # type: ignore[misc]
-    @classmethod
```

Comment:
Otherwise it throws an error `TypeError: 'classmethod' object is not callable`.
Maybe you can reproduce it locally when reverting and running the tests.

Do you have a better solution?